pkgs/development/libraries/dbus/default.nix

```diff
@@ -68,25 +52,26 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" "lib" "doc" "man" ];
 
+  strictDpes = true;
   nativeBuildInputs = [
     autoreconfHook
     autoconf-archive
     pkg-config
     docbook_xml_dtd_44
     docbook-xsl-nons
     xmlto
-  ];
+  ] ++ lib.optional enableSystemd systemd;
```

Comment:
Won’t this cause build systemd to be used instead of host one in the service file?

Reply:
yeah, i now defined it manually like the docs say https://cgit.freedesktop.org/dbus/dbus/tree/configure.ac#n1027